content/en/docs/openshift/install-kubeflow.md

```diff
@@ -95,7 +95,7 @@ Use the following steps to install Kubeflow 1.0 on OpenShift 4.x.
     ...
     ```
 
-1. The command below looks up the URL of Kubeflow user interface assigned by the OpenShift cluster. You can open the printed URL in your broser to access the Kubeflow user interface.
+1. The command below looks up the URL of Kubeflow user interface assigned by the OpenShift cluster. You can open the printed URL in your browser to access the Kubeflow user interface.
```

Comment:
Thanks @kelvins Just a small change suggestion - add `the` (definite article) to `Kubeflow user interface`:

```suggestion
1. The command below looks up the URL of the Kubeflow user interface assigned by the OpenShift cluster. You can open the printed URL in your browser to access the Kubeflow user interface.
```